config/routes.rb

```diff
@@ -1,12 +1,10 @@
-# frozen_string_literal: true
-
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root "users#index"
 
-  # Defines the root path route ("/")
-  # root "articles#index"
-  # root "users#index"
-  resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show]
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show, :create, :new] do
+      resources :comments, only: [:create, :new]
+      post 'like', on: :member
```

Comment:
- [ ] You should also modify your routes to accommodate the Likes controller and model you will be creating which I have suggested earlier.